mycore-restapi/src/main/resources/components/restapi/config/mycore.properties

```diff
@@ -9,6 +9,9 @@ MCR.RestAPI.v1.Files.URL.path=servlets/MCRFileNodeServlet/${derid}/
 MCR.RestAPI.v1.Filter.Write.IPs.Pattern=127\\.0\\.0\\.1|::1
 MCR.RestAPI.v1.Filter.Read.IPs.Pattern=\\.*
 
+# For filtering xml metadata set this property with your filter stylesheet
+# MCR.RestAPI.v1.Filter.XML=xsl/mycoreobject-xml.xsl
```

Comment:
`MCR.RestAPI.v1.Filter.XML=mycoreobject-xml` would use the mycoreobject-xml Content-Transformer (if defined)